<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>learnwindow</class>
 <widget class="QDialog" name="learnwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nauka słówek</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>201</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="plWordLabel">
      <property name="text">
       <string>Polskie słowo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="engWordLabel">
      <property name="text">
       <string>Angielskie słowo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="synonymLabel">
      <property name="text">
       <string>Synonimy:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="sentenceLabel">
      <property name="text">
       <string>Zdanie:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="checkButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>270</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Sprawdź</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>239</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="buttonsLayout">
    <item>
     <widget class="QPushButton" name="incorrectlyButton">
      <property name="text">
       <string>Nie wiem</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="almostButton">
      <property name="text">
       <string>Prawie</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="correctlyButton">
      <property name="text">
       <string>Wiem</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="learnedWordsLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Nauczone słowa</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="newWordsLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Na dziś słów: </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
